frontend-react/package.json

```diff
@@ -93,6 +91,9 @@
             "import/newline-after-import": 1,
             "import/no-commonjs": 0,
             "import/no-named-as-default": 0,
+            "prettier/prettier": "error",
+            "arrow-body-style": "off",
+            "prefer-arrow-callback": "off",
```

Comment:
These ESLint rules are problematic with the Prettier plugin so we turn them off. See https://github.com/prettier/eslint-plugin-prettier#arrow-body-style-and-prefer-arrow-callback-issue.